.travis/test.sh

```diff
@@ -25,5 +25,15 @@ fi
 export OMERO_SERVER_VERSION=5.4
 export OMERO_WEB_VERSION=5.4
 export COMPOSE_FILE="docker-compose.yml:volumes.yml"
+
+if [ ! -d ".omeroci" ];
+then
+  mkdir .omeroci
+  echo "#!/bin/sh" > .omeroci/test-data
```

Comment:
https://github.com/openmicroscopy/omero-test-infra/blob/master/docker#L186 is missing an `|| echo ignore`

Reply:
@joshmoore I just saw your PR in [openmicroscopy/omero-test-infra](https://github.com/openmicroscopy/omero-test-infra/pull/10), thanks for doing that! I've made a temporary commit using those changes just to ensure they work on travis. And they do (yay!).